### ISSUE TITLE
ContentExists makes messages hard to parse

### ISSUE THREAD

**Issue by mengelbart** [moq-wg/moq-transport#404]
#343 and #389 introduced a `ContentExists` field to SUBSCRIBE_OK, SUBSCRIBE_FIN, and SUBSCRIBE_RESET. If I read the notation correctly, it has a size of 1 bit, which makes it hard to parse the rest of the message. Could we encode that information in the message type by using two types per message instead of a 1-bit field?

**Comment by afrind** [comment]
I think the intent is that it is one *byte* not one bit.

**Comment by suhasHere** [comment]
or just a byte field with only 2 possible values allowed (1) and (0) ..

**Comment by fluffy** [comment]
I agree we need to keep this byte aligned however we do it.

**Comment by fluffy** [comment]
Have a look at PR #405
